hive/utils/rliable_matrix.py

```diff
@@ -0,0 +1,188 @@
+import argparse
+import os
+import pickle as pkl
+import matplotlib.patches as mpatches
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+from matplotlib.ticker import MaxNLocator
+from rliable import library as rly
+from rliable import metrics, plot_utils
+from scipy.stats.stats import find_repeats
+
+
+def score_normalization(score_, min_scores, max_scores):
+    norm_scores = (score_.copy() - min_scores) / (max_scores - min_scores)
+    return norm_scores
+
+
+def convert_to_matrix(score_dict):
+    keys = sorted(list(score_dict.keys()))
+    return np.stack([score_dict[k] for k in keys], axis=1)
+
+
+def load_and_read_rewards(
+    agent_name, env_names, path, min_env_score, max_env_score, normalization
+):
+    score_dict = {}
+    for env_index, env_name in enumerate(env_names):
+        path_temp = os.path.join(path, env_name, agent_name)
+        seed_s = [i for i in os.listdir(path_temp) if i[0] != "."]
+        score_ = []
+        for seed in seed_s:
+            job_files = [
+                i for i in os.listdir(os.path.join(path_temp, seed)) if i[0] != "."
+            ]
+            for job_file in job_files:
+                pkl_file = open(
+                    os.path.join(
+                        path_temp,
+                        seed,
+                        job_file,
+                        "logger/logger_0/log_data.p",
```

Comment:
You should not assume you have this file structure. For now, write the functions in the following way:
`process_rewards(rewards)`, where rewards is whatever format/structure you want (ie assume the data has already been loaded for you. Then write a separate function to read the rewards. For this, you can use the functions in `visualization.py` if that's easier (eg something like `find_all_experiments_data()` and you just write code to pass the appropriate arguments.